pkg/gator/errors.go

```diff
@@ -51,9 +51,6 @@ var (
 	// object or oldObject for the underlying framework to review.
 	// This mimicks the k8s api server behvaior.
 	ErrNoObjectForReview = errors.New("no object or oldObject found to review")
-	// ErrNilOldObject indicates that the AdmissionRequest did not provide an oldObject.
-	// Gatekeeper expects oldObject to be non nil on DELETE operations.
-	ErrNilOldObject = errors.New("oldObject is nil")
```

Comment:
Thanks for the PR!

Isn’t it better to provide the details that oldObjec is missing?

Reply:
Fair question but the way we reworked the UX now that pointers is still well communicated I think:

```
$ gator verify test/gator/verify
    --- FAIL: system-user       (0.002s)
        not a valid AdmissionReview object: oldObject cannot be nil for DELETE operations
FAIL
...
```

wdyt?